manual/source/Infrastructure/Manage/RepositoryStructure.rst

```diff
@@ -26,7 +26,7 @@ SeqAn supports the usual CMake build types and we recommend :ref:`using multiple
 In addition to ``CMAKE_BUILD_TYPE`` there is also the ``SEQAN_BUILD_SYSTEM`` parameter which can be one of
 
 #. ``DEVELOP`` -- all build targets (apps, demos, tests) and documentation (dox, manual) are created (the default).
-#. ``SEQAN_RELEASE_LIBRARY`` -- only dox and libraray target created.
+#. ``SEQAN_RELEASE_LIBRARY`` -- only dox and library targets are created.
```

Comment:
OK. It's an option. That PR add "SEQAN_NO_DOX". So the default mode contains dox like before.